### PR TITLE
INREL-5132: alter the paragraph form to limit display modes

### PR DESCRIPTION
**Pull request by killes**

### CODE REVIEW

File: modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module [hunk 1]
```diff
@@ -236,3 +236,27 @@ EOT;
...
   return $cssPlaceholder;
 }
+
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function burdastyle_thunder_admin_field_widget_paragraphs_form_alter(&$element, &$form_state, $context) {
+  if ($element['#paragraph_type'] == 'content_reference') {
+    foreach ($element['subform']['field_e_view_mode']['widget']['#options'] as $key => $value) {
```
Comment (mgoedecke) [inline modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module:249]: check if $element['subform']['field_e_view_mode']['widget']['#options'] is set

File: modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module [hunk 2]
```diff
@@ -236,3 +236,27 @@ EOT;
...
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_preprocess_views_view().
+ */
+function burdastyle_thunder_admin_preprocess_views_view(&$variables) {
+  if ($variables['view']->id() == 'content_reference_browser') {
```
Comment (mgoedecke) [inline modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module]: check if $variables['view'] is actually a view. We had similar issues with $variables['node'] being a Drupal\Entity\Node most of the time, but in an edge case it was just a nodeid

File: modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module (hunk 1 above)
Comment (killes) [reply modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module:249]: done

File: modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module (hunk 2 above)
Comment (killes) [reply modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module]: done

**Pull request merged**